web-ui/src/main/resources/catalog/js/admin/HarvestController.js

```diff
@@ -3,677 +3,126 @@
 
 
 
-
-
-  goog.require('gn_category');
   goog.require('gn_harvester');
-  goog.require('gn_importxsl');
+  goog.require('gn_harvest_settings_controller');
+  goog.require('gn_harvest_report_controller');
 
   var module = angular.module('gn_harvest_controller',
-      ['gn_harvester', 'gn_category', 'gn_importxsl']);
+      ['gn_harvest_settings_controller',
+       'gn_harvest_report_controller', 'gn_harvester']);
 
 
   /**
    *
    */
   module.controller('GnHarvestController', [
-    '$scope', '$http', '$translate', '$injector', '$rootScope',
-    'gnSearchManagerService', 'gnUtilityService',
-    function($scope, $http, $translate, $injector, $rootScope,
-             gnSearchManagerService, gnUtilityService) {
+    '$scope', '$http', 'gnUtilityService',
+    function($scope,$http,gnUtilityService) {
+		$scope.isLoadingHarvester = false;
```

Comment:
Would be cool at some point to use jslint to preserve formatting. See https://github.com/geonetwork/core-geonetwork/tree/develop/web-ui